### ISSUE TITLE
Cita doesn't support proceedings

### ISSUE THREAD

**Issue by Futur3r** [diegodlh/zotero-cita#224]
**Describe the bug**
When synchronizing the references of an item, cita doesn't sync proceedings and shows an error 

**To Reproduce**
Sync the citations of an item that cites proceedings, for example "[Géographie, anthropologie et anarchie au XIXe siècle](https://www.wikidata.org/wiki/Q115627780)"

**Expected behavior**
Sure, there is no "proceedings" type in Zotero, but the reference could be sync with the "book" type and a tag in it's extra fields like "Type: proceedings" ?

**Environment:**
 - Zotero version: 6.0.13
 - Cita version: 0.5.3-beta
 - OS name and version: Gentoo linux

**Comment by diegodlh** [comment]
Hi, @Futur3r! Thanks for your report.

Zotero doesn't have a "proceedings" item type, but does have a "conference paper" item type, which maps to Wikidata's "conference paper" ([Q23927052](https://www.wikidata.org/wiki/Q23927052)), defined as "document published as part of the proceedings of an academic conference".

From this, it looks like a "proceedings" would be a collection of "conference papers". And it would make sense that a work cites a conference paper in a proceedings, instead of the entire proceedings.

Does this make sense? Shouldn't these cites work targets in Wikidata you mention (such as Q115703103) be changed to instances of "conference paper" instead?

**Comment by Futur3r** [comment]
Yes, it should, but the author cited the proceedings and not a single conference paper in these proceedings.
I'm just extracting data from the "references" section of the paper. Same for a book, an author can cite an entire book without having to cite a specific book section. It's almost impossible for me to know which specific paper an author cites, the only person who knows this is the author him⋅herself.

And it's not the only one, I found that kind of citations sometimes and it's trickier when there is no Open Access version of the papers.

So we could adapt Cita to the nature of the data ?

**Comment by diegodlh** [comment]
Sorry for my delayed replies.

Because of how Cita works (how the citation lists are saved, how cited items can be linked to items in the Zotero library, etc), I would not support cited item types which are not item types in Zotero.

However, I don't mean we should continue ignoring cited items of types not supported. Instead, we may use a higher class which includes the type not supported. For example, for the "proceedings" case, we may use the generic "document" type. Issue #93 is closely related to this.

So, if in Wikidata it says an item cites another of type "proceedings", Cita would import it as a citation of type "document". The other way round, if one adds a citation in Cita of type "document", it may be uploaded to Wikidata as a "cites work" statement to an element of type "work", and the user may then further refine this in Wikidata.

**Comment by Futur3r** [comment]
Yes, having a default item type is better. I think Zotero use also the 'document' type as a default sometimes.
It will be easier to code, as a simple 'else' statement.
